apps/ios/Shared/Views/Helpers/SwipeLabel.swift

```diff
@@ -0,0 +1,73 @@
+//
+//  SwipeLabel.swift
+//  SimpleX (iOS)
+//
+//  Created by Levitating Pineapple on 06/08/2024.
+//  Copyright © 2024 SimpleX Chat. All rights reserved.
+//
+
+import SwiftUI
+
+struct SwipeLabel: View {
+    private let text: String
+    private let systemImage: String
+    private let inverted: Bool
+
+    init(_ text: String, systemImage: String, inverted: Bool) {
+        self.text = text
+        self.systemImage = systemImage
+        self.inverted = inverted
+    }
+
+    var body: Image {
+        Image(
+            uiImage: SwipeActionView(
+                systemName: systemImage,
+                text: text
+            ).snapshot(inverted: inverted)
+        )
+    }
+
+    private class SwipeActionView: UIView {
+        private let imageView = UIImageView()
+        private let label = UILabel()
+
+        init(systemName: String, text: String) {
+            super.init(frame: CGRect(x: 0, y: 0, width: 64, height: 48))
+            imageView.image = UIImage(systemName: systemName)
+            imageView.contentMode = .scaleAspectFit
+            label.text = NSLocalizedString(text, comment: "swipe action")
+            label.textAlignment = .center
+            addSubview(imageView)
+            addSubview(label)
+        }
+
+        override func layoutSubviews() {
+            imageView.frame = CGRect(
+                x: 18,
+                y: 0,
+                width: 28,
+                height: 28
+            )
+            label.frame = CGRect(
+                x: 0,
+                y: 32,
+                width: 64,
+                height: 16
+            )
+        }
+
+        @available(*, unavailable)
+        required init?(coder: NSCoder) { fatalError("not implemented") }
```

Comment:
looks scary, why do we not need it?

Reply:
maybe better to return some empty rectangle here?

Reply:
@spaced4ndy 
This is called when you initialise from a storyboard.
We don't use storyboards.